frontend/src/views/PageNotFound.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <v-container fluid>
+    <content-card
+      class="contentCard"
+      :title="$tc('views.pageNotFound.title')"
+      max-width="1200"
+      toolbar
+    >
+      <template #title-actions>
+        <v-btn
+          class="d-md-none"
+          icon
+          :to="{ name: 'profile', query: { isDetail: true } }"
+        >
+          <user-avatar :user="user" :size="36" />
+        </v-btn>
+      </template>
+
+      <v-card-text class="justify-center">
+        <p id="error-title" class="font-weight-bold text-center px-8 text-h4">
+          {{ $tc('views.pageNotFound.detail') }}
+        </p>
+        <v-img contain class="error-image" src="/public/fourZeroFour.svg"></v-img>
+      </v-card-text>
+
+      <v-card-actions class="justify-center">
+        <icon-button
+          v-if="hasHistory"
+          class="mr-5"
+          icon="mdi-arrow-left"
+          @click="$router.go(-1)"
+          >{{ $tc('views.pageNotFound.back') }}</icon-button
+        >
+        <icon-button icon="mdi-tent" @click="$router.push({ name: 'camps' })">{{
+          $tc('views.pageNotFound.goToCamps')
+        }}</icon-button>
+        <!-- This doesnt work for some reason: -->
+        <v-icon size="32">$vuetify.icons.fourZeroFour</v-icon>
```

Comment:
I tried to use v-icon like the camp image does but I didn't get it to work

Reply:
For me this gets shown next to the buttons:
<img width="379" alt="Bildschirmfoto 2023-06-03 um 15 11 53" src="https://github.com/ecamp/ecamp3/assets/3001985/df033b87-e5fc-42d5-a2a2-48527b20c00f">

But I think I would recommend the way I suggested above.